src/_scss/pages/agencyLanding/table/cells/_agencyLandingCell.scss

```diff
@@ -0,0 +1,19 @@
+@mixin agencyLandingCell() {
+    .cell-content {
+        padding: rem(12) rem(16);
+        color: $color-base;
+        box-sizing: border-box;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
```

Comment:
This causes strange behavior for smaller screens:
<img width="528" alt="screen shot 2017-07-31 at 2 44 19 pm" src="https://user-images.githubusercontent.com/3594363/28792779-c0bb21a0-75fe-11e7-963a-858581f1d390.png">